public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -126,6 +127,10 @@ export class EhrViewComponent implements OnInit, OnDestroy {
   private searchCallback(results: any) {
     this.searchResult = results;
     this.items = this.searchResult.items;
+    this.standardConcepts = this.searchResult.standardConcepts;
+    if (this.searchResult.matchType) {
```

Comment:
Is matchType sometimes not set? Could we just use searchResult.matchType in our template and not need the separate variable? For items and standard concepts i wanted them to be empty arrays and wasn't sure if the api would return that for each.

Reply:
Yeah, changed it to use the searchResult.matchType.